administrator/components/com_content/models/article.php

```diff
@@ -318,6 +318,18 @@ public function getItem($pk = null)
 			}
 		}
 
+		$menu = JFactory::getApplication()->getMenu('site');
+		$menuItems = $menu->getItems('link', 'index.php?option=com_content&view=article&id=' . (int) $item->id);
+
+		if(!empty($menuItems)) {
```

Comment:
I checked the code through phpcs... I will again check the code to check phpcs error... Btw did you test that the code solves the issue?